scipy/special/add_newdocs.py

```diff
@@ -2263,7 +2326,42 @@ def add_newdoc(place, name, doc):
     """
     expm1(x)
 
-    exp(x) - 1 for use when `x` is near zero.
+    Compute ``exp(x) - 1 ``, for use when `x` is near zero.
```

Comment:
Sounds like a good idea.  Feel free to suggest better wording. :)

I probably won't get back to this for a day or so.  When I do, I'll fix the doctest failure and the wording.